openssl-sys/Cargo.toml

```diff
@@ -23,7 +24,7 @@ libc = "0.2"
 bssl-sys = { version = "0.1.0", optional = true }
 
 [build-dependencies]
-bindgen = { version = "0.65.0", optional = true, features = ["experimental"] }
+bindgen = { version = "0.69.0", optional = true, features = ["experimental"] }
```

Comment:
Do we still need the experimental feature?

Reply:
Yes, not static-fns are not yet non-experimental. (I'll file a ticket about that if there's not one).